gnome/libgit2-glib/Portfile

```diff
@@ -1,29 +1,29 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           meson 1.0
+PortGroup           gitlab 1.0
+PortGroup           meson  1.0
 
-# upgrade to version 1.0.0+ requires glib2 >= 2.68
+# Upgrade to version 1.0.0.1+ requires glib2 >= 2.68. Currently MacPorts using
+# glib 2.66.8, which is missing g_time_zone_new_identifier() and
+# g_string_replace().
+# See https://gitlab.gnome.org/GNOME/libgit2-glib/-/issues/76
```

Comment:
Given this requirement, will everything be fully-functional without `glib2` being at 2.68+?

Reply:
Version 1.0.0 only needs `g_string_replace()` from glib 2.68+, which uses in the `translate_windows_paths()` function for fix in the Windows. libgit2-glib can be built without this feature, which is not used in macOS. So we can upgrade to version 1.0.0.

Version 1.0.0.1+ needs `g_time_zone_new_identifier()` from glib 2.68+. I don't know in which function libgit2-glib uses this function, but it is part of the program itself, without which it will not work correctly.